components/raftstore/src/store/fsm/apply.rs

```diff
@@ -1432,9 +1433,22 @@ where
         for req in requests {
             let cmd_type = req.get_cmd_type();
             let mut resp = match cmd_type {
-                CmdType::Put => self.handle_put(ctx.kv_wb_mut(), req),
-                CmdType::Delete => self.handle_delete(ctx.kv_wb_mut(), req),
+                CmdType::Put => {
```

Comment:
Just counting the QPS at apply module can't cover all scenarios, there is a big different CPU cost  in grpc and raftstore between 1 request put 100kv and 100 reqeusts with each request puts 1 kv.

Reply:
emmm, maybe statistics should be placed at a higher level?